client/src/game/ui/menu/menu.vue

```diff
@@ -196,6 +213,22 @@ export default class MenuBar extends Vue {
     openDmSettings(): void {
         EventBus.$emit("DmSettings.Open");
     }
+
+    delMarker(marker: string): void {
+        gameStore.removeMarker({ marker: marker, sync: true });
+    }
+
+    jumpToMarker(marker: string): void {
+        gameStore.jumpToMarker(marker);
+    }
+
+    nameMarker(marker: string): string {
+        if (layerManager.UUIDMap.get(marker) != undefined) {
+            return layerManager.UUIDMap.get(marker)!.name;
+        } else {
+            return "";
```

Comment:
This should not occur anymore since there's either a marker or there isn't. I can't think of a situation where you get a return but not know what. The undefined check is more to satisfy TS, I'm certain you won't get undefined back from the server.

Reply:
That's fine and good to hear that it shouldn't occur in normal situations.

I would still prefer the [?] to be returned here instead of in the template though.